docs/ops/signals/DFT_7.md

```diff
@@ -0,0 +1,148 @@
+## DFT <a name="DFT"></a> {#openvino_docs_ops_signals_DFT_7}
+
+**Versioned name**: *DFT-7*
+
+**Category**: Signal processing
+
+**Short description**: *DFT* layer performs the discrete complex-to-complex Fourier transformation of input tensor by specified dimensions.
+
+**Attributes**:
+
+    No attributes available.
+
+**Inputs**
+
+*   **1**: `data` - Input tensor of type *T* with data for the DFT transformation. Type of elements is any supported floating point type. The last dimension of the input tensor must be equal to 2, i.e. the input tensor shape must have the form `[D_0, D_1, ..., D_{N-1}, 2]`, representing the real and imaginary components of complex numbers in `[:, ..., :, 0]` and in `[:, ..., :, 1]` correspondingly.  Required.
+*   **2**: `axes` - 1D tensor of type *T_IND* specifying dimension indices where DFT is applied, and `axes` is any unordered list of indices of different dimensions of input tensor, e.g. `[0, 4]`, `[4, 0]`, `[4, 2, 1]`, `[1, 2, 3]`. These indices should be non-negative integers from `0` to `rank(data) - 1` inclusively.  Other dimensions do not change. The order of elements in `axes` attribute matters, and mapped directly to elements in the 3d input `signal_size`. Required.
```

Comment:
axes can't have the same values in different places, right? Why we don't want to use only sorted array here?

Reply:
We don't want to use only sorted array, because PyTorch can use unsorted array of axes. See functions `resize_fft_input` (started at https://github.com/pytorch/pytorch/blob/master/aten/src/ATen/native/SpectralOps.cpp#L71) and `canonicalize_fft_shape_and_dim_args` (started at https://github.com/pytorch/pytorch/blob/master/aten/src/ATen/native/SpectralOps.cpp#L190).